### PR TITLE
Add more allowed qml module versions and remove ones that don't exist

### PR DESCRIPTION
**Pull request by pvuorela**
@jpetrell @jlehtoranta

### CODE REVIEW

File: allowed_qmlimports.conf [hunk 1]
```diff
@@ -11,15 +11,23 @@ QtQuick.Window 2.0
 # Qt 5.1
```
Comment (martyone) [inline allowed_qmlimports.conf]: worth to update this as well? :)

Comment (pvuorela) [inline allowed_qmlimports.conf]: How?

Comment (martyone) [inline allowed_qmlimports.conf]: Forget it - I missed the context above this line :)

**Comment by jpetrell** [comment]
Add "import QtQuick.Window 2.0" and "import QtQuick 2.3|4|5|6"?

**Comment by pvuorela** [comment]
> Add "import QtQuick.Window 2.0" 

Already there.

> and "import QtQuick 2.3|4|5|6"?

Not supported yet.

**Comment by jpetrell** [comment]
Oh this is taken into use before Qt 5.6. So no Qt 5.6 QtQml.Models 2.2, QtQml.StateMachine 1.0, Qt.labs.settings 1.0, etc..

Looks like import QtQml 2.0 and 2.1 are missing?

**Comment by pvuorela** [comment]
Yea, now first adding modules that should have been there long time already.

QtQml should indeed be there too. Adding.

**Comment by pvuorela** [comment]
There.

**Comment by jpetrell** [comment]
LGTM

**Pull request merged**